terraform/layer1-aws/aws-cloudtrail.tf

```diff
@@ -8,10 +9,18 @@ resource "aws_cloudtrail" "main" {
 
   tags = local.tags
 }
-
+#tfsec:ignore:aws-s3-enable-bucket-logging tfsec:ignore:aws-s3-enable-versioning tfsec:ignore:aws-s3-encryption-customer-key
 resource "aws_s3_bucket" "cloudtrail" {
   bucket = "${local.name}-aws-cloudtrail-logs"
 
+  server_side_encryption_configuration {
```

Comment:
Based on Terraform documentation, this parameter is deprecated. It's recommended to use [aws_s3_bucket_server_side_encryption_configuration](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket_server_side_encryption_configuration)

Reply:
corrected